app/controllers/access_requests_controller.rb

```diff
@@ -1,5 +1,20 @@
 class AccessRequestsController < ApplicationController
   def index
     @access_requests = AccessRequest.unapproved.order(request_date_utc: :asc)
+
+    @access_request_id = params[:access_request_id]
```

Comment:
I think we can use the flash to simplify the logic and make the view a lot simpler.

Reply:
Did something about this 👍